agile_problem2.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Problem2 do
+	def largest_2_numbers(), do: -2

```

Comment:
Same comment as for problem 1, + same comment for the usage of a arbitrary number instead of an atom.